src/components/ListTitle.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { ReactElement } from 'react'
+import { SvgProps } from 'react-native-svg'
+import styled from 'styled-components/native'
+
+const Container = styled.View`
+  min-height: 54px;
+  align-items: center;
+  margin-bottom: 32px;
+  margin-top: 32px;
+`
+
+const ScreenTitle = styled.Text`
+  line-height: 18px;
```

Comment:
I think line-height is not needed here?

Reply:
It also messes up the title.
Without line-height:
![Bildschirmfoto zu 2021-10-21 08-47-34](https://user-images.githubusercontent.com/18463686/138225835-0bd2a0a6-045c-470e-8491-2e267801b3a6.png)
With line-height:
![Bildschirmfoto zu 2021-10-21 08-47-02](https://user-images.githubusercontent.com/18463686/138225841-ea6aaad2-1d75-4aee-88b7-30db340ea4fa.png)